2386/2/Counters.rb

```diff
@@ -0,0 +1,31 @@
+require 'russian_obscenity'
+
+# Class with count logic =)
+class Counters
+  PATH_FOLDER = 'Rapbattle'.freeze
```

Comment:
it's better to create an instance of counter and pass battles to initializer and call all these methods